cmd/update-notifier.go

```
@@ -28,25 +28,35 @@ import (
 	"github.com/fatih/color"
 )
 
+// computeUpdateMessage - calculates the update message, only if a
+// newer version is available.
+func computeUpdateMessage(downloadURL string, oldVersion, newVersion time.Time) (string, bool) {
+	if !newVersion.After(oldVersion) || downloadURL == "" {
+		return "", false
+	}
+	// Last argument below is never actually used as (newVersion >
+	// oldVersion).
+	newerThanStr := humanize.RelTime(oldVersion, newVersion, "ago", "huh?")
```

This `huh?` is not nice.. We need to find something else. In-fact when this happens we can throw an error i.e `return "", false` which kind of makes sense to me that no update is needed.. It can happen so that users can build it themselves using `make release` .

---

> This huh? is not nice.. We need to find something else. In-fact when this happens we can throw an error i.e return "", false which kind of makes sense to me that no update is needed.. It can happen so that users can build it themselves using make release .

I think then it is better to switch back to `humanize.Time()`, since we are explicitly handling the "relative" aspects of the time. We are already returning from the function when `newVersion <= oldVersion`, so the `huh?` string is never used. Let me amend this patch.

---

My bad, can't use `humanize.Time()` as that is relative to the current time.

So I just made it an empty string - there is a comment explaining that the 4th argument is never used, so I think this is ok.